### ISSUE TITLE
Error on line 171 - Decision Trees

### ISSUE THREAD

**Issue by jacksusank** [rtealwitter/datamining2026spring#23]
I think this statement is misleading/false:

Old Line: "Recall that the gradient descent update subtracts the gradient of the parameter (opposite the direction of steepest ascent) for a small learning rate α."

Suggested fix: "Recall that gradient descent updates parameters by subtracting the gradient of the loss with respect to those parameters (opposite the direction of steepest ascent) for a small learning rate α."

https://github.com/rtealwitter/datamining2026spring/blob/4ab71393f59e05fd1ad92b1a427756007888f03a/notes/DecisionTrees.qmd#L171

**Comment by rtealwitter** [comment]
Yours is more specific but both seem correct to me
